agcaballero/portfolio/src/main/webapp/blog.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
```

Comment:
2020?

Reply:
@ykjchen I'll go change that. I just copied and pasted all the licenses from the original script.js file, and I didn't want to mess with the legalities.

Reply:
Done